kolibri/content/migrations/0001_initial.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
```

Comment:
I thought we had agreed to avoid adding migrations to the repo during development?

@MCGallaspy @benjaoming thoughts?